scripts/helpers/lock-editing.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
please with for all functions

return type, usage and output

like in other helpers
https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/editor/props.js